src/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { Component } from "@angular/core";
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormsModule } from "@angular/forms";
+import { By } from "@angular/platform-browser";
+import { BreadcrumbComponent } from "./breadcrumb.component";
+import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
+
+@Component({
+	selector: "test-cmp",
+	template: `
+	<ibm-breadcrumb>
+		<ibm-breadcrumb-item href="./home">
+			Home
+		</ibm-breadcrumb-item>
+	</ibm-breadcrumb>`,
+	entryComponents: [BreadcrumbComponent]
+})
+class BreadcrumbComponentTestComponent { }
+
+describe("BreadcrumbComponent", () => {
+	let component: BreadcrumbComponent;
+	let fixture: ComponentFixture<BreadcrumbComponent>;
+
+	beforeEach(async(() => {
+		TestBed.configureTestingModule({
+			imports: [FormsModule],
+			schemas: [CUSTOM_ELEMENTS_SCHEMA],
+			declarations: [
+				BreadcrumbComponent,
+				BreadcrumbItemComponent,
+				BreadcrumbComponentTestComponent
+			],
+		}).compileComponents();
+	}));
+
+	beforeEach(() => {
+		fixture = TestBed.createComponent(BreadcrumbComponent);
+		component = fixture.componentInstance;
+		fixture.detectChanges();
+	});
+
+	it("should create", () => {
```

Comment:
it should be `"should work"` to be consistent with other components. It also reads better if it fails :)